### DISCUSSION TITLE
Sandpack for support live code preview and editing

### DISCUSSION THREAD

**Discussion by bang9** [GitbookIO/community#148] [Feature Requests]
### What's your feature/idea?

Hi

This is a [link](https://react.dev/learn/tutorial-tic-tac-toe#what-are-you-building) to the official documentation newly released in React.

Here, there is an editor that allows you to edit React code live and see the results immediately.
This feature uses the [Sandpack](https://github.com/codesandbox/sandpack) library supported by CodeSandbox.

Can Gitbook also support this type of component?

### What's your desired outcome?

Use sandpack live editor as a block, like `CodePen` embed

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

**Comment by CashVo** [comment]
FYI - Gitbook support the RunKit integration that does this:
https://docs.gitbook.com/product-tour/integrations/runkit

**Reply by bang9** [reply]
Thank you for the information. But it's different from my use case.

**Comment by MantisClone** [comment]
@bang9 I was able to successfully embed a Browser CodeSandBox. Unfortunately, embedding a newer Cloud CodeSandBox doesn't seem to be supported. See #211 for details.
